src/XML/XMLValidationServiceCollection.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace CultuurNet\UDB3\UDB2\XML;
+
+class XMLValidationServiceCollection
+{
+    /**
+     * @var XMLValidationServiceInterface[]
+     */
+    private $xmlValidationServices;
+
+    /**
+     * XMLValidationServiceCollection constructor.
+     * @param XMLValidationServiceInterface[] $xmlValidationServices
+     */
+    public function __construct(XMLValidationServiceInterface... $xmlValidationServices)
```

Comment:
Aangezien je hier al met variadic arguments werkt, stel ik voor om deze class te laten vallen en in de constructor van CombinedXmlValidationService gewoon met variadic arguments te werken.